services/tenant-ui/config/default.json

```diff
@@ -8,6 +8,11 @@
     "showInnkeeperAdminLogin": true,
     "showOIDCReservationLogin": false,
     "showWritableComponents": true,
+    "tokenExpirySeconds": "86400",
+    "session": {
+      "timeoutSeconds": "600",
+      "countdownSeconds": "10"
+    },
```

Comment:
These can (probably should) be added to the environment configuration and then propagated through helm charts. Unless we want to just leave them as hard coded defaults... which for this feature wouldn't be that bad as it's not really something that would probably change too much per environment. (though the expiry might vary actually)

That can definitely be done in a subsequent PR after we try this out, unless you're really keen to take a crack at it (I can show you where things are defined etc), but if so definitely want to rebase in all of Ivan's latest PR as all the helm chart files are recently changed locations.

Reply:
I added these variable at 10 mins activity session and 30 seconds countdown for all environments.